web/pdf_find_controller.js

```diff
@@ -533,12 +554,14 @@ class PDFFindController {
 
               for (let j = 0, jj = textItems.length; j < jj; j++) {
                 strBuf.push(textItems[j].str);
+                if (textItems[j].hasEOL) {
+                  strBuf.push("\n");
+                }
               }
 
               // Store the normalized page content (text items) as one string.
-              [this._pageContents[i], this._pageDiffs[i]] = normalize(
-                strBuf.join("")
-              );
+              const query = strBuf.join("");
+              [this._pageContents[i], this._pageDiffs[i]] = normalize(query);
```

Comment:
Looks like an unrelated change, and the `query` variable name looks strange here (since it's actually page-contents); please just revert these changes.